packages/website/pages/docs/usage/messages.mdx

```diff
@@ -149,6 +149,7 @@ t('selectordinal', {year: 11});
 ## Rich text
 
 You can format rich text with custom tags and map them to React components.
+If you want to use the same tag multiple times, you can configure it via the [default translation values](/docs/usage/configuration#default-translation-values).
```

Comment:
Nit: Can we move this after the code snippet? I feel like it could be helpful to first finish introducing the concept and then providing supplemental information like the global config. It might also help a bit with discovery, I tend to skip directly to the code examples when reading a section and expect advanced information to follow afterwards.